src/message.rs

```diff
@@ -132,8 +132,9 @@ pub(crate) fn mismatch(expected: &str, actual: &str) {
     println!();
 }
 
-pub(crate) fn output(warnings: &str, output: &Output) {
+pub(crate) fn output(warnings: &str, output: &Output, build_out: &str) {
```

Comment:
I would prefer that the caller concatenate the build output and run output and pass only one Output, so these are printed together under one STDOUT heading. It will be sufficiently clear which parts come from the build and which parts from the compiled code, and this matches what the author would see from `cargo run`.